tests/test_failsafe.py

```diff
@@ -108,10 +108,10 @@ def test_circuit_breaker(self):
         failing_operation = create_failing_operation()
 
         with pytest.raises(CircuitOpen):
-            policy = RetryPolicy(5, [SomeRetriableException])
+            policy = ExceptionHandlingPolicy(5, [SomeRetriableException])
             circuit_breaker = CircuitBreaker(maximum_failures=2)
             loop.run_until_complete(
-                Failsafe(retry_policy=policy, circuit_breaker=circuit_breaker)
+                Failsafe(exception_handling_policy=policy, circuit_breaker=circuit_breaker)
                 .run(failing_operation)
             )
 
```

Comment:
Can you please add tests checking that:

1) Exception was raised as expected when `abortable_exceptions` is set (similar to what we have in test_fallback_failsafe.py)
2) Failure wasn't reported to circuit breaker when abortable exception is thrown